runtime/src/bank.rs

```diff
@@ -3985,13 +3985,17 @@ impl Bank {
     // multiple times with the same parent_slot in the case of forking.
     //
     // Generally, all of sysvar update granularity should be slot boundaries.
-    fn get_sysvar_account(&self, pubkey: &Pubkey) -> Option<Account> {
+    fn get_sysvar_account(&self, pubkey: &Pubkey, is_root_fixed: bool) -> Option<Account> {
         let mut ancestors = self.ancestors.clone();
         ancestors.remove(&self.slot());
-        self.rc
-            .accounts
-            .load_slow(&ancestors, pubkey)
-            .map(|(acc, _slot)| acc)
+        let res = if is_root_fixed {
```

Comment:
Maybe, my prev round of comment was confusing, but I don't think this condition is needed because this fn's sole caller is just always giving us `true`.

Reply:
So, I'm saying this time that you can directly call `self.rc.accounts.load_slow_with_fixed_root(...)` from here.

Reply:
hmmm I was slightly worried somebody might call `get_sysvar_account` in the future in a nonsafe way,  if there wasn't some indication that it may be potentially dangerous. Maybe I should just rename this function to `get_sysvar_account_with_fixed_root`?